smriprep/workflows/outputs.py

```diff
@@ -73,137 +77,600 @@ def init_anat_reports_wf(*, freesurfer, output_dir, name="anat_reports_wf"):
+                output_names=["before", "after"],
+                input_names=["mask_file", "before", "after", "after_mask"],
+            ),
+            name="norm_msk",
+        )
+        norm_rpt = pe.Node(SimpleBeforeAfter(), name="norm_rpt", mem_gb=0.1)
+        norm_rpt.inputs.after_label = "Participant"  # after
+
+        ds_std_t1w_report = pe.Node(
+            DerivativesDataSink(
+                base_directory=output_dir, suffix="T1w", datatype="figures"
+            ),
+            name="ds_std_t1w_report",
+            run_without_submitting=True,
+        )
+
+        # fmt:off
+        workflow.connect([
+            (inputnode, template_iterator_wf, [
+                ("template", "inputnode.template"),
+                ("anat2std_xfm", "inputnode.anat2std_xfm"),
+            ]),
+            (inputnode, t1w_std, [("t1w_preproc", "input_image")]),
+            (inputnode, mask_std, [("t1w_mask", "input_image")]),
+            (template_iterator_wf, t1w_std, [
+                ("outputnode.anat2std_xfm", "transforms"),
+                ("outputnode.std_t1w", "reference_image"),
+            ]),
+            (template_iterator_wf, mask_std, [
+                ("outputnode.anat2std_xfm", "transforms"),
+                ("outputnode.std_t1w", "reference_image"),
+            ]),
+            (template_iterator_wf, norm_rpt, [('outputnode.space', 'before_label')]),
+            (t1w_std, norm_msk, [('output_image', 'after')]),
+            (mask_std, norm_msk, [('output_image', 'after_mask')]),
+            (template_iterator_wf, norm_msk, [
+                ('outputnode.std_t1w', 'before'),
+                ('outputnode.std_mask', 'mask_file'),
+            ]),
+            (norm_msk, norm_rpt, [
+                ('before', 'before'),
+                ('after', 'after'),
+            ]),
+            (inputnode, ds_std_t1w_report, [('source_file', 'source_file')]),
+            (template_iterator_wf, ds_std_t1w_report, [('outputnode.space', 'space')]),
+            (norm_rpt, ds_std_t1w_report, [('out_report', 'in_file')]),
+        ])
+        # fmt:on
+
+    if freesurfer:
+        from ..interfaces.reports import FSSurfaceReport
+
+        recon_report = pe.Node(FSSurfaceReport(), name="recon_report")
+        recon_report.interface._always_run = True
+
+        ds_recon_report = pe.Node(
+            DerivativesDataSink(
+                base_directory=output_dir, desc="reconall", datatype="figures"
+            ),
+            name="ds_recon_report",
+            run_without_submitting=True,
+        )
+        # fmt:off
+        workflow.connect([
+            (inputnode, recon_report, [('subjects_dir', 'subjects_dir'),
+                                       ('subject_id', 'subject_id')]),
+            (recon_report, ds_recon_report, [('out_report', 'in_file')]),
+            (inputnode, ds_recon_report, [('source_file', 'source_file')])
+        ])
+        # fmt:on
+
+    return workflow
+
+
+def init_ds_template_wf(
```

Comment:
I feel like we need a better name, but this is the subject template, not MNI or other template spaces.